src/aura/STG_CMP_Tabs/STG_CMP_Tabs.cmp

```diff
@@ -31,50 +31,25 @@
-                    <aura:set attribute="else">
-                        <ui:outputURL aura:id="coursesTabBtn" value="#" label="{!$Label.c.stgTabCourse}" click="{!c.tabNavigationClick}" class="coursesTab course-con-menu-item" />
-                    </aura:set>
-                </aura:if>
+                <ui:outputURL aura:id="coursesTabBtn" value="#" label="{!$Label.c.stgTabCourse}" click="{!c.tabNavigationClick}" class="coursesTab course-con-menu-item" />
             </li>
             <li aura:id="courseConTab" id="courseConTab" class="slds-tabs__item slds-text-heading--label" title="Course Connections" role="tab">
-                <aura:if isTrue="{!v.namespacePrefix == 'hed__'}">
-                    <ui:outputURL aura:id="courseConTabBtn" value="#" label="{!$Label.hed.stgTabCourseConn}" click="{!c.tabNavigationClick}" class="courseConTab course-con-menu-item" />                    
-                    <aura:set attribute="else">
-                        <ui:outputURL aura:id="courseConTabBtn" value="#" label="{!$Label.c.stgTabCourseConn}" click="{!c.tabNavigationClick}" class="courseConTab course-con-menu-item" />                    
-                    </aura:set>
-                </aura:if>
+                <ui:outputURL aura:id="courseConTabBtn" value="#" label="{!$Label.c.stgTabCourseConn}" click="{!c.tabNavigationClick}" class="courseConTab course-con-menu-item" />
             </li>
             <li aura:id="addrTab" id="addrTab" class="slds-tabs__item slds-text-heading--label" title="Accounts and Contacts" role="tab">
-                <aura:if isTrue="{!v.namespacePrefix == 'hed__'}">
-                    <ui:outputURL aura:id="addrTabBtn" value="#" label="{!$Label.hed.stgTabAccCont}" click="{!c.tabNavigationClick}" class="addrTab accs-cons-menu-item" />
-                    <aura:set attribute="else">
-                        <ui:outputURL aura:id="addrTabBtn" value="#" label="{!$Label.c.stgTabAccCont}" click="{!c.tabNavigationClick}" class="addrTab accs-cons-menu-item" />
-                    </aura:set>
-                </aura:if>
+                <ui:outputURL aura:id="addrTabBtn" value="#" label="{!$Label.c.stgTabAccCont}" click="{!c.tabNavigationClick}" class="addrTab accs-cons-menu-item" />
             </li>
             <li aura:id="systemTab" id="systemTab" class="slds-tabs__item slds-text-heading--label" title="System" role="tab">
                 <aura:if isTrue="{!v.namespacePrefix == 'hed__'}">
-                    <ui:outputURL aura:id="systemTabBtn" value="#" label="{!$Label.hed.stgTabSystem}" click="{!c.tabNavigationClick}" class="systemTab system-menu-item" />                    
+                    <ui:outputURL aura:id="systemTabBtn" value="#" label="{!$Label.hed.stgTabSystem}" click="{!c.tabNavigationClick}" class="systemTab system-menu-item" />
```

Comment:
@xinnanlisf Do we still need this, or can we just use the second link? 

It looks like a number of components still have 'hed__' references actually: 
- STG_CMP_Addr.cmp
- STG_CMP_Container.cmp
- STG_CMP_Rel.cmp
- STG_CMP_System.cmp
- STG_CMP_Tabs.cmp

Reply:
Hey @lparrott, good catch! These should be good now.